submissions/tests/test_models.py

```diff
@@ -164,3 +163,39 @@ def test_highest_score_hidden(self):
         highest = ScoreSummary.objects.get(student_item=item).highest
         self.assertEqual(highest.points_earned, 1)
         self.assertEqual(highest.points_possible, 2)
+
+    def test_override_highest_no_score(self):
+        item = StudentItem.objects.create(
```

Comment:
We're creating this same item in multiple locations. Could we consolidate into either a helper method or a test case `setUp`?